gremlin-util/src/test/java/org/apache/tinkerpop/gremlin/structure/io/AbstractTypedCompatibilityTest.java

```diff
@@ -882,27 +883,35 @@ public void shouldReadWriteOutDirection() throws Exception {
     }
 
     @Test
-    public void shouldReadWriteVarBulkSet() throws Exception {
-        final String resourceName = "var-bulkset";
+    @Ignore("re-enable after GraphSONV4 bulklist is implemented")
```

Comment:
Technically, you could keep this enabled for GraphBinary and just add a skip in the GraphSON test file since this is just an abstract test class. It's small enough that it doesn't need to be updated for this PR.